compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -850,7 +850,7 @@ trait Implicits:
     && !to.isError
     && !ctx.isAfterTyper
     && ctx.mode.is(Mode.ImplicitsEnabled)
-    && from.isValueType
+    && from.widen.isValueType
```

Comment:
Counter-example:

```scala
  given foo[A]: (Ctx => Int) = _ => 42

  val works = get[Int](using summon[Ctx => Int])
  val fails = get[Int]
```

using the example from #16453 - with this change (or `!imp.underlying.isInstanceOf[PolyType]` in `ignoredConvertibleImplicits`) the suggestion of `foo` will be dropped, even though it's a correct suggestion.